hoot-cmd/src/main/cpp/hoot/cmd/main.cpp

```diff
@@ -90,12 +90,6 @@ int main(int argc, char *argv[])
       return -1;
     }
   }
-  // Yep, a juvenile easter egg. :)
-  else if (QString(argv[1]) == "whoami")
-  {
-    cout << "Papa Smurf" << endl;
-    return 0;
-  }
   else
```

Comment:
I feel that we've reached a level of maturity where we don't need this anymore...who am I kidding we probably haven't, but I removed it anyway :-)